contributing.md

```diff
@@ -43,7 +50,8 @@ git pull upstream master
 git checkout -b features/123-boolean-operators
 ```
 
-* Commit locally as you progress (`git add` and `git commit`) Use a properly formatted commit message, write tests that fail before your change and pass afterward, run all the tests locally and in parallel for different process counts (`mpirun -np <PROCESSES>`). Be sure to document any changed behavior in docstrings, keeping to HeAT's docstring standard (explained below).
+* Commit locally as you progress (`git add` and `git commit`) Use a properly formatted commit message, write tests that fail before your change and pass afterward, run all the tests locally and in parallel for different process counts (`mpirun -np <PROCESSES>`). Be sure to document any changed behavior in docstrings, keeping to HeAT's [docstring standard](https://github.com/helmholtz-analytics/heat/blob/504-docstring-formatting/doc/source/documentation_howto.rst).
```

Comment:
I like the link to the docstrings... can we mention that again further down under "guidelines" ? (line 100 )

Reply:
Yes good idea, done